frontend/src/metabase/ui/components/overlays/Modal/Modal.styled.tsx

```diff
@@ -1,12 +1,18 @@
 import type { MantineThemeOverride } from "@mantine/core";
+import cx from "classnames";
 import { t } from "ttag";
 
-// See `zIndex` prop at https://v6.mantine.dev/core/modal/?t=props
-export const DEFAULT_MODAL_Z_INDEX = 200;
+import Animation from "metabase/css/core/animation.module.css";
+import ZIndex from "metabase/css/core/z-index.module.css";
+
 const DEFAULT_MODAL_SPACING = "lg";
 
 export const getModalOverrides = (): MantineThemeOverride["components"] => ({
   Modal: {
+    classNames: {
+      overlay: cx(ZIndex.Overlay, Animation.fadeIn),
+      inner: cx(ZIndex.Overlay, Animation.popInFromBottom),
```

Comment:
Is the animations introduced in this PR, or has it always been there?

Reply:
I introduced them in this PR, since the new approach to stacking requires us to use CSS transitions to animate overlays